core/src/main/java/io/cucumber/core/event/EventBus.java

```diff
@@ -7,6 +7,8 @@
 
     Long getTime();
 
+    Long getElapsedTimeMillis();
```

Comment:
Same here, but `getTimeStampMillis`.

Reply:
its the elapsed time since epoch in millis so i named it that way. But i ll update the name no problem.